addons/frag/functions/fnc_getMaterialInfo.sqf

```diff
@@ -0,0 +1,62 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * This function returns a classification of material type based on the surface hit.
+ *
+ * Arguments:
+ * 0: Surface type given as either a CfgSurfaces path or .bisurf filepath, same format as "HitPart" projectile parameter <STRING>
+ *
+ * Return Value:
+ * Material categories as expanded on in line 44 below <STRING>
+ *
+ * Example:
+ * [_surfaceType] call ace_frag_fnc_getMaterialInfo;
```

Comment:
```suggestion
 * _surfaceType call ace_frag_fnc_getMaterialInfo
```
Drop the trailing `;` in header examples. Also would be better if _surfaceType was an actual value that could be used to call the function (so you can just copy-paste-run the example and get a valid return).

Reply:
Alright, sounds good, I'll get them all cleaned up

Reply:
Alright removed some magic numbers I missed and reformatted all the examples. I also removed excess brackets on function calls that weren't need, per your changes here